pyproject.toml

```diff
@@ -17,7 +17,7 @@ starlette = "^0.32.0"
 requests = "^2.32.3"
 uvicorn = {extras = ["standard"], version = "^0.30.3"}
 pydantic = {extras = ["email"], version = "^1.10.17"}
-SQLAlchemy = ">=1.4.51,<1.5"
+SQLAlchemy = ">=1.4.51,<2.1"
```

Comment:
Should we jump to requiring SQLAlchemy 2+?

Reply:
[7793b06](https://github.com/mozilla-it/ctms-api/pull/924/commits/7793b06ed62ffa4c5047439020bf0e7011c3f887)